lib/locreator.js

```diff
@@ -12,6 +13,16 @@ var Locreator = function (nemo) {
   this.drivex = Drivex(nemo.driver, nemo.wd);
   this.locatex = Locatex(nemo);
   this.normify = this.normalize = function (locator) {
+    if(!locator.locator || !locator.locator.trim()) {
+      error('NO or EMPTY locator found, please fix it');
```

Comment:
I was thinking about it but then I remembered our previous conversation not to throw errors from plugins. I don't entirely recall the reasoning behind it, but I vote for throwing an error. Let me know what you think